src/app/common/stores/applications/Distribution/Wallet.store.ts

```diff
@@ -68,20 +75,72 @@ class WalletStore extends BaseStore implements Wallet {
    */
   initWalletHooks() {
     if (this.onboardSdk) return;
-    const connectionConfig = {
-      dappId: config.ONBOARD.API_KEY,
-      networkId: 1,
-      walletSelect: {
-        wallets,
+    const injected = injectedModule();
```

Comment:
Code duplication. Requested to refactor in different PR.

Reply:
https://blox-ensemble.monday.com/boards/4755515647/pulses/5522174407

we have task for this